package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "module4a",
+  "version": "1.0.0",
+  "description": "For this assignment, you're going to fork this repository to your own Github account and clone that repository locally. Inside the repo you'll find a placeholder for app.js, a placeholder trucks.js, and a 'public' directory. Inside the public directory you'll find an index.html file and a client.js file.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
Your package.json doesn't have any dependencies listed in it. When you application has to require additional dependencies, you want to add them to the package.json file by using the --save switch when installing them yourself.